flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/ClusterHealthEvaluator.java

```diff
@@ -97,25 +97,28 @@ public void evaluate(
                 LOG.debug("Last valid health info: {}", lastValidClusterHealthInfo);
                 LOG.debug("Observed health info: {}", observedClusterHealthInfo);
 
-                boolean isHealthy =
+                ClusterHealthResult jobHealthResult =
```

Comment:
use `var`

Reply:
Updated to use `var`

Side note: Just for my own learning, why do we prefer to use `var`?